schools-schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE schools (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    address TEXT NOT NULL,
+    sf_district TEXT NOT NULL,
+    students TEXT,
```

Comment:
I modeled the data types (e.g. string vs. numeric) off of the data in our old school data file, but was definitely feeling kind of iffy about some of them in db form. I'll go head and change this to numeric. 

Do you think the field name should be changed as well (like "num_students" or students_enrolled), or are we good with "students"?

Reply:
I changed the data type to "numeric" (should it be "integer"?) and updated the seed file so that the data input for students enrolled is now numeric. I changed any empty string values for this field to NULL.